code/game/gamemodes/clock_cult/clock_scripture.dm

```diff
@@ -934,8 +932,8 @@ Judgement: 10 servants, 100 CV, and any existing AIs are converted or destroyed
 	desc = "Creates a mania motor which will cause brain damage and hallucinations in nearby non-servant humans. It will also try to convert humans directly adjecent to the motor."
 	invocations = list("Znl guv'f genafzvggr-e...", "...oernx gur jvyy-bs nyy jub bccbfr hf!")
 	channel_time = 80
-	required_components = list("guvax_capacitor" = 4, "replicant_alloy" = 1, "hierophant_ansible" = 1)
-	consumed_components = list("guvax_capacitor" = 3, "replicant_alloy" = 1, "hierophant_ansible" = 1)
+	required_components = list("guvax_capacitor" = 5, "belligerent_eye" = 3, "replicant_alloy" = 1, "hierophant_ansible" = 1)
+	consumed_components = list("guvax_capacitor" = 4, "belligerent_eye" = 2, "replicant_alloy" = 1, "hierophant_ansible" = 1)
```

Comment:
Consider upping the power costs of these if they're too strong, components aren't the only thing you can use to balance.